src/components/molecules/SpaceEditFormNG/SpaceEditFormNG.tsx

```diff
@@ -219,7 +288,13 @@ export const SpaceEditFormNG: React.FC<SpaceEditFormNGProps> = ({
           </>
         </AdminSpacesListItem>
         <AdminSpacesListItem title="Appearance" isOpened>
-          <AdminSection title="Upload a banner photo">
+          <AdminSection title="Default portal appearence">
+            <PortalVisibility
+              updateRoomVisibility={updateRoomVisibility}
+              visibilityState={room?.visibility}
+            />
```

Comment:
Same comments like previous uses of this component.

Reply:
reef https://github.com/sparkletown/sparkle/pull/2449#discussion_r742024970